manual/content/point-data/point-data.md

```diff
@@ -0,0 +1,349 @@
+:::
+
+To limit the pixel size, add the `EnablePixelSizeControl` and `MaxPixelSize` settings in the `SizeSettings` table:
+
+```lua
+  ...
+  Renderable = {
+    Type = "RenderablePointCloud",
+    File = asset.resource("path/to/dataset.csv"),
+    SizeSettings = {
+      -- Limit the size of the points to a max size, in pixels
+      MaxPixelSize = 4.7,
+      EnablePixelSizeControl = true
+    }
+  },
+  ...
+```
+
+At the core, the size of the points is still determined by the exponential scaling. That is, as long as the camera is far enough away so that the points do not exceed the specified pixel size, the size will still be determined by the provided scale exponent. Also, the multiplicative `ScaleFactor` is applied after the pixel size scaling. That is, if you specify a `MaxPixelSize` of 5 and a `ScaleFactor` of 2, the max size of the points will be 2 * 5 = 10 pixels.
+
+:::{note}
+The pixel-based scaling currently only works for planar display systems. In other setups, it might lead to discontinuities between views and incorrect scaling at the edges.
+:::
+
+### Scale Based on Data
+
+It is also possible to do some simple scaling based on data parameters. The size of the points is then scaled based on the value in the data column.
+
+Similarly to the color mapping, the size mapping is added by specifying a list of options that can be used for setting the size of the points:
+
+```lua
+  ...
+  Renderable = {
+    Type = "RenderablePointCloud",
+    File = asset.resource("path/to/dataset.csv"),
+    SizeSettings = {
+      -- The options for the columns that the points can be scaled by. The first
+      -- alternative is chosen per default
+      SizeMapping = { "one parameter", "another parameter" },
+    }
+  },
+  ...
+```
+:::{attention}
+The size mapping is currently a bit of an experimental feature. For now, the point size if directly multiplied with the data value is directly multiplied by the data value of the chosen `SizeMapping` column. This is not always suitable though, depending on the range of the data values. The behavior may be subject to change in the future.
+:::
+
+### Summary of Final Size Computation
+
+In summary, the order of which the settings affect the size of the points is the following:
+
+1. 10 ^ `ScaleExponent` * Scale From Data => world scale size
+2. Limit pixel size scale => prevent the points from growing larger than a certain screen space size
+3. Finally, multiply with `ScaleFactor` to increase or decrease the size, onscreen
+
+## Fading
+
+A point cloud can also be set up so that it fades in and out based on the distance to the camera. The distance is computed based on the origin of the dataset.
```

Comment:
You might also say this is recommended like limiting the pixel size above, or maybe just mention the problem that this can solve, which is a visual overload of points.

Reply:
Good point.  I'll add a sentence about why/when this may be useful